cli/flox/src/utils/dialog.rs

```diff
@@ -14,6 +16,13 @@ pub struct Select<T> {
     pub options: Vec<T>,
 }
 
+pub struct Spinner<T>(T);
```

Comment:
```suggestion
pub struct Spinner<F>(F);
```
Just to make it clear that it holds the closure and not the type the closure returns

Reply:
Makes sense 👍🏼